proteus/Domain.py

```diff
@@ -17,6 +19,17 @@ def __init__(self,nd,name="defaultDomain",units="m"):
         self.units=units
         self.x=[]#minx,miny,minz
         self.L=[]#bounding box when self.x is origin
+        self.g=g
```

Comment:
@tridelat can you explain why you're attaching gravity to the domain? Generally we're trying to separate the geometry from the other  physical parameter inputs, so I might want to find a way to pass gravity where you need  it without adding to the domain classes.